### PR TITLE
Add types for unofficial-builds

### PR DESCRIPTION
**Pull request by wangling12**
https://github.com/nodejs/unofficial-builds/issues/97

### CODE REVIEW

**Comment by rvagg** [comment]
I think we need to add linux-x64-glibc-217 too; looking at https://unofficial-builds.nodejs.org/download/release/v20.6.0/ and what we're currently publishing that seems to be the only other missing one

**Comment by wangling12** [comment]
> I think we need to add linux-x64-glibc-217 too; looking at https://unofficial-builds.nodejs.org/download/release/v20.6.0/ and what we're currently publishing that seems to be the only other missing one

Done, It really looks like a temporary test name. :joy:

File: README.md [hunk 1]
```diff
@@ -36,7 +36,9 @@ OS / architecture / packaging specifiers used in the listings include:
 * **linux-arm64**: normally .tar.gz and .tar.xz (also known as AArch64 or ARMv8)
 * **linux-armv6l**: normally .tar.gz and .tar.xz
 * **linux-armv7l**: normally .tar.gz and .tar.xz
+* **linux-loong64**: normally .tar.gz and .tar.xz
 * **linux-ppc64le**: normally .tar.gz and .tar.xz
+* **linux-riscv64**: normally .tar.gz and .tar.xz
 * **linux-s390x**: normally .tar.gz and .tar.xz
 * **linux-x64**: normally .tar.gz and .tar.xz
```
Comment (rvagg) [inline README.md:43]:
```suggestion
* **linux-x64**: normally .tar.gz and .tar.xz
* **linux-x64-glibc-217**: normally .tar.gz and .tar.xz
```

Comment (wangling12) [reply README.md:43]: Done

**Review by rvagg — APPROVED** [review]

**Pull request merged**